src/settings/__tests__/settings.browser.test.mjs

```diff
@@ -98,6 +98,83 @@ describe('Settings in Browser test suite', () => {
       expect(items).toContainEqual({label, icon});
     });
   });
+  describe('Pane Titles', () => {
+    describe('Services', () => {
+      test('title contains icon', () => {
+        // Services pane is shown by default
```

Comment:
This is a big assumption since other tests can change the initial state.
Ensure the Services pane is visible just like you're doing in the other scenarios.